modules/tinymce/src/core/test/ts/browser/undo/UndoKeyboardShortcutsTest.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyHooks } from '@ephox/wrap-mcagar';
```

Comment:
```suggestion
import { PlatformDetection } from '@ephox/sand';
import { TinyAssertions, TinyContentActions, TinyHooks } from '@ephox/wrap-mcagar';
```
We have plans for a lint rule for this, but we haven't implemented it yet sorry.